dashboard/api.py

```diff
@@ -1 +1,92 @@
 import requests
+
+def get_visit_events(project_name, after_time, limit, after_id=None)
```

Comment:
add default values for limit (10) and after_time (30 days prior to current date)